opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -167,6 +167,72 @@ enum BatchMessage {
 
 /// A [`LogProcessor`] that buffers log records and reports
 /// them at a pre-configured interval from a dedicated background thread.
+// **Memory Management in BatchLogProcessor**
```

Comment:
a lot of these can be moved off to docs.rs itself, as they are good for users to be aware of.